cmd/helm-sync/main.go

```diff
@@ -141,11 +155,15 @@ func main() {
 				os.Exit(1)
 			}
 
+			step := backoff.Step()
+
 			failCount++
```

Comment:
This has some implications on the duration that this process will error loop, since it will exit once `failCount >= *flMaxSyncFailures`. This is because the sleep duration will be less for the first dozen or so iterations (until it reaches the cap). 

I think this is probably still a reasonable duration to spend error looping before exiting, so I don't expect this should cause any problems. WDYT @nan-yu?

Reply:
Agreed. I don't think the abnormal exit duration matters that much if the helm-sync container is already in a failure state.

Reply:
SG, thanks for confirmation